Src/Plugins/Security/SecurePlaintextSecretsValidators/GitHubTokenValidationHelper.cs

```diff
@@ -0,0 +1,165 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Sdk;
+
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
+{
+    internal class GitHubTokenValidationHelper
+    {
+        protected virtual string[] GitHubEndpoints => new[]
```

Comment:
>GitHubEndpoints [](http://example.com/codeflow?start=35&length=15)

Make the GitHubEndpoint virtual so in the internal plugin it can be overridden with additional endpoints to support verification against GH enterprise and MS internal endpoints.